### ISSUE TITLE
Talk Page: About Us

### ISSUE THREAD

**Issue by kdecks** [RetroAchievements/docs-archive#31]
## short history
The short history makes me feel a little uncomfortable. There's years of history gap, and plenty happened during that time. As written it feel like first the site was started, and then things started happening after 2017.

I don't have a proposed solution yet.

## staff

- [ ] I intend to match the the staff data to how it is in the welcome channel, and then remove that section after it's done.

**Comment by meleu** [comment]
Agreed, that page is not yet on the docs (it doesn't appear in the Sidebar here in the wiki).

I've just dumped what came to my mind and it would be cool if someone could put more info there (but not forgetting that it is a **short** history).

**Comment by kdecks** [comment]
### About us
Scott updated the section. Which feels better. I've removed the point about me starting the Discord server. There's just so many great noteworthy projects made by devs and admin before and since, that I feel I don't deserve the recognition unless all, or many more of these got recognition too. Also, I started the Discord server with the help especially of @GameDragon2k and @ScottFromDerby.

**Comment by meleu** [comment]
The [About Us](https://docs.retroachievements.org/About-Us) page is already alive, IMO the staff info on the #welcome channel can be replaced with a simple link to that page.
